civihr_default_theme/scss/components/_leave-calendar.scss

```diff
@@ -0,0 +1,122 @@
+$chr-leave-calendar-cell-height:                  35px;
+$chr-leave-calendar-cell-font-size:               10px;
+$chr-leave-calendar-item-requested-border-color:  #c7cbd0;
+$chr-leave-calendar-item-requested-border-height: 0.5em;
```

Comment:
`em` depends on font-size. Are we sure we want to do that? and the border should not have a fixed height?